rpcs3/Emu/RSX/VK/VKProgramPipeline.cpp

```diff
@@ -534,26 +536,28 @@ namespace vk
 				case input_type_texel_buffer:
 				{
 					auto &buffer_view = texel_buffer_views[texel_buffer_index];
-					buffer_view = null_buffer_view();
 
 					auto &buffer = texel_buffers[texel_buffer_index++];
-					buffer.buffer = null_buffer();
+					buffer.buffer = VK_NULL_HANDLE;
```

Comment:
You cant pass VK_NULL_HANDLE to AMD drivers, hence the null_buffer descriptor. You'd have to throw an exception to keep the driver from crashing.